.github/workflows/build.yml

```diff
@@ -70,7 +74,8 @@ jobs:
       - uses: actions/checkout@v2
       - name: setup
         run: |
-          sudo apt install libxkbcommon-x11-0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 libxcb-xinerama0 libxcb-xfixes0
+          sudo apt-get update
+          sudo apt install libxkbcommon-x11-0 libxcb-icccm4 libxcb-image0 libxcb-keysyms1 libxcb-randr0 libxcb-render-util0 libxcb-xinerama0 libxcb-xfixes0 libegl-dev -y
```

Comment:
why -dev?

Reply:
This fixes an import error related to Qt:
```
ImportError: libEGL.so.1: cannot open shared object file: No such file or directory
```